source/blender/editors/space_buttons/buttons_context.c

```diff
@@ -608,6 +608,7 @@ static bool buttons_context_path(
     case BCONTEXT_OBJECT:
     case BCONTEXT_PHYSICS:
     case BCONTEXT_CONSTRAINT:
+    case BCONTEXT_GAME:
```

Comment:
I'd move it that with the other case: which have a break; to avoid "fallthrough" I think

Reply:
Maybe I'm missing something as I haven't looked much into what `buttons_context_path_object` actually does. But the fallthrough was intentional since I thought the excution path for `BCONTEXT_GAME` should be handled in a similar manner for the case with `BCONTEXT_OBJECT` because an `Object` is where `GameObjectSettings` instance is attached to.

Please let me know if that's not a correct behaviour.

Reply:
yes, indeed, looks good after looking at it in visual studio 👍